src/test/java/de/adorsys/sdjwt/sdjwtvp/SdJwtVPTest.java

```diff
@@ -186,4 +189,21 @@ public void testS6_2_PresentationPartialDisclosure() throws ParseException, JOSE
                 .verifySignature(TestSettings.verifierContextFrom(presenteSdJwtVP.getCnfClaim(), "ES256"));
     }
 
-}
+    @Test
+    public void testOf_validInput() {
+    String sdJwtString = TestUtils.readFileAsString(getClass(), "sdjwt/s6.2-presented-sdjwtvp.txt");
+    SdJwtVP sdJwtVP = SdJwtVP.of(sdJwtString);
+
+    assertNotNull(sdJwtVP);
+    assertEquals(4, sdJwtVP.getDisclosures().size());
+    }
```

Comment:
Indent the body of the function. When you'll switch to IntelliJ, I'll show you how you can autoformat your changes locally, without affecting the rest of the file.

Reply:
Okay !
Thank you.